src/kex/kex.h

```diff
@@ -21,7 +21,6 @@
 
 enum OQS_KEX_alg_name {
```

Comment:
We should remove kex.h

Reply:
I'll remove it in issue: https://github.com/open-quantum-safe/liboqs/issues/372